sdk/appcenter-push/src/main/java/com/microsoft/appcenter/push/Push.java

```diff
@@ -173,7 +173,11 @@ public static void checkLaunchedFromNotification(Activity activity, Intent inten
      * Sender ID of your project before starting the Push service.
      *
      * @param senderId sender ID of your project.
+     *
+     * @deprecated Starting Android P at release date and later all versions of Android after April 2019,
+     * Firebase SDK is required to use push. Please follow the migration guide at https://aka.ms/acfba.
```

Comment:
This link shows how to remove firebase dependencies, not add them.  I don't know if we have documentation for this yet.

Reply:
See @guperrot's comment above https://github.com/Microsoft/AppCenter-SDK-Android/pull/744#discussion_r198592561